### DISCUSSION TITLE
add button or shortcut for "My Discussions"

### DISCUSSION THREAD

**Discussion by prudhomm** [community/community#2994] [Discussions] [answered]
In the search bar it is possible to retrieve our own discussions. 
it would be nice to have a more direct way (button? next to Answered/Not Answered) to access the discussions we have authored or are involved with.

**Comment by becca** [comment, marked as answer]
Does https://docs.github.com/en/free-pro-team@latest/discussions/guides/finding-discussions-across-multiple-repositories#finding-discussions help you find your discussions?

**Reply by prudhomm** [reply]
It certainly helps, but it doesn't seem right to have this menu item there.
